scripts/reformat-source

```diff
@@ -32,5 +32,5 @@ cd "$SOURCE" || {
 	exit 1
 }
 
-find . \( \( -name '*.[ch]' -or -name '*.[ch]pp' -or -name '*.[ch].in' \) -and -not -regex '^./src/tools/gen.*' \) \
-	-exec "$CLANG_FORMAT" -style=file -i {} +
+source_files=$(git ls-files '*.c' '*.h' '*.cpp' '*.hpp' '*.c.in' '*.h.in' | grep -v '^src/tools/gen.*')
+printf "%s\n" "$source_files" | sed -nE 's/.*/"\0"/p' | xargs "$CLANG_FORMAT" -style=file -i
```

Comment:
Unfortunately this command does not work on macOS as far as I can tell:

```sh
reformat-source 2>&1 | grep 'No such file or directory' | wc -l | sed -E 's/[ ]+//g'
#> 1026
```

. The problem seems to be `sed -nE 's/.*/"\0"/p'`, which prints `"0"` 1026 😊 times on my machine. If you want to add surrounding `"` chars around each file name, then the following should work:

```
sed -nE 's/(.*)/"\1"/p'
``` 

. I **do not think** that this is a particular safe method to work with the filenames, since it does not handle special characters properly. For our needs it should be enough though, as long as we do not add any source files with names that need escaping.

Reply:
I now use `sed -nE 's/(.*)/'"'"'\1'"'"'/p'` which should surround filenames with `'`. That should be safer.